homeworks/andrii.soloviov_Solowey-A/8-components-templates/my-app/src/app/app.component.html

```diff
@@ -0,0 +1,4 @@
+<div class="container">
+  <h1 class="head-line">{{ appTitle}}</h1>
```

Comment:
```suggestion
  <h1 class="head-line">Gallery-app</h1>
```
Якщо `appTitle` не є динамічним то і робити для нього інтерполяцію не потрібно